src/components/visualizations/chart/highcharts/test/comboConfiguration.spec.ts

```diff
@@ -39,7 +76,7 @@ describe("Combo Configuration", () => {
         });
     });
 
-    it("should disable saturation on other line series in line_line combo chart?", () => {
+    it("should disable saturation on other series in combo which have line series on both axis", () => {
```

Comment:
--> `should disable saturation on other series in combo which has line series on both axes`